api.yaml

```diff
@@ -83,7 +85,14 @@ paths:
           async: false
           request: '#! json .Request.Body !#'
           success: '{"status": 200, "bodyPassthrough": true}'
-          error: '{"status": 500, "bodyPassthrough": true}'
+          error: >
```

Comment:
Not sure if this requires an Error object section at the end of the yaml file but here is where I placed logic for the function to return a 400 if the Config message is missing a mandatory value

Reply:
I believe there should be an error object, sort of like how asset-inventory-api handles invalid input:
```
400:
          description: "Invalid input"
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/Error'
```
 and under schemas at the bottom of the file:
```
    Error:
      type: object
      properties:
        errorMessage:
          type: string
        errorType:
          type: string
        stackTrace:
          type: array
          items:
            type: string
```

Reply:
@dscgarcia I think errorType comes from serverfull https://github.com/asecurityteam/serverfull/blob/c9f5f135e262f0006372c036fba72f71a4d43299/invokeapi.go#L47

Reply:
awesome! perfect, thank you for finding this. will add it now